src/plugins/intel_cpu/src/emitters/tpp/x64/kernel_executors/brgemm.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) 2020-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "brgemm.hpp"
+#include "transformations/tpp/x64/op/brgemm.hpp"
+
+using namespace Xbyak;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64;
+
+namespace ov {
+namespace intel_cpu {
+#define COMPILE_BRGEMM_TPP_KERNEL(...)                                        \
```

Comment:
Can we reuse a similar macros from jit_tpp_emitter instead of creating a new one?

Reply:
done in [668504d](https://github.com/openvinotoolkit/openvino/pull/28304/commits/668504db46e95358a60def59715496475d6b0f86), thanks Vladislav!